viewer/src/main/java/nl/tailormap/viewer/stripes/ProxyRESTActionBean.java

```diff
@@ -121,7 +145,7 @@ private URL constructURL() throws MalformedURLException {
         URL requestUrl = new URL(context.getRequest().getRequestURL().toString());
         String host = context.getServletContext().getInitParameter("flamingo.restproxy.host");
         String port = context.getServletContext().getInitParameter("flamingo.restproxy.port");
-        String constructedURL = "http://" + (host != null ? host : "localhost") + ":" + (port != null ? port : "8084") + "/feature-api" + url + parentId;
+        String constructedURL = "http://" + (host != null ? host : "localhost") + ":" + (port != null ? port : "8084") + "/" + endpoints.get(endpoint) + url + parentId;
```

Comment:
If the `endpoint` parameter is not in the map this will proxy to `http://localhost:8084/null`? Returning 404 seems better